R/estimate_ascertainment.R

```diff
@@ -84,7 +77,14 @@ estimate_ascertainment <- function(data,
   )
   checkmate::assert_int(burn_in, lower = 0)
   checkmate::assert_date(max_date, null.ok = TRUE)
-  checkmate::assert_class(epidist, "epidist", null.ok = TRUE)
+
+  # check delay_density and run over short sequence to test output
+  stopifnot(
+    "`delay_density` must be a distribution density function with 1 argument\\
+    evaluating density at a vector of values and returning a numeric vector.\\
+    E.g. function(x) stats::dgamma(shape = 5, scale = 1, x = x)" =
```

Comment:
It's probably clearer to keep the default order of arguments here (flagging just once but impacts repeated code in the codebase):

```suggestion
    E.g. function(x) stats::dgamma(x = x, shape = 5, scale = 1)" =
```

Reply:
Fixed in https://github.com/epiverse-trace/cfr/pull/88/commits/0117f2a3885e69a040d28c091c64a2d425b2db2f